terraform/ec2-examples/distributed-ml-training/main.tf

```diff
@@ -128,6 +128,11 @@ module "autoscaling_head" {
   }
 
   tags = local.tags
+
+  metadata_options = {
```

Comment:
just curious, but why? (would be great to add on PR for future reference of why this change was warranted/needed)

Reply:
Medium risk reported by a checkov scan: https://docs.prismacloud.io/en/enterprise-edition/policy-reference/aws-policies/aws-general-policies/bc-aws-general-31